idb/postgres/postgres.go

```diff
@@ -2117,9 +2120,12 @@ func (db *IndexerDb) yieldAccountsThread(req *getAccountsRequest) {
 						Total:         ap.Total,
 						Decimals:      uint64(ap.Decimals),
 						DefaultFrozen: boolPtr(ap.DefaultFrozen),
-						UnitName:      stringPtr(ap.UnitName),
-						Name:          stringPtr(ap.AssetName),
-						Url:           stringPtr(ap.URL),
+						UnitName:      stringPtr(ap.AssetParams.UnitName),
+						UnitNameB64:   baPtrOrDefault(ap.UnitNameBytes, []byte(ap.AssetParams.UnitName)),
```

Comment:
This will result in []byte{} where we would rather have nil? If UnitName is "", `[]byte(ap.AssetParams.UnitName)` will be `[]byte{}`, but wouldn't we rather have nil?

Reply:
Fixed